installers/python-ci/radiasoft-download.sh

```diff
@@ -50,7 +50,7 @@ python_ci_main() {
                 pip install git+https://'${GITHUB_TOKEN:+$GITHUB_TOKEN@}'github.com/radiasoft/\$x.git
             done
             pip uninstall -y '$r' >& /dev/null || true
-            pip install -e .
+            pip install --no-cache-dir -e .
```

Comment:
Leave a comment saying why we have this. It will be in the git log but I think this is so specific and non-standard (for us) that it is good to have a comment too. Something like:
```
# GitHub CI runners are limited on disk space so use --no-cache-dir to limit disk usage as much as
# possible. See git.radiasoft.org/downloads/issues/XXX
```

Reply:
90d140f5a24c095c86312cd89cff22be1f11900c